### DISCUSSION TITLE
Allow exporting as CSV directly to file system

### DISCUSSION THREAD

**Discussion by MyNameIsRomayne** [iSoron/uhabits#1838] [Feature Requests]
Choosing export as CSV asks for what apps to do it, but I really just want to have it export to somewhere on my phone, like Downloads, so that I can upload it to my computer and put it in Google Sheets that way or whatnot. I don't ever want to export my data to Discord, Bluetooth, or E-Mail, though it just so happens that E-Mail is currently a workaround for that.

(I'd love to PR this but I don't know how to code in the language this app is in. If it's easy enough then I could? I guess?)

**Comment by turkerozturk** [comment]
You can install Total Commander app, and then send your CSV to its icon, it will ask you to access to your local storage, read its privacy policy(it is ok), accept it, then browse the path where you want to export. I tried it, it is ok.
